README.md

```diff
@@ -74,6 +74,8 @@ After selecting the items wanted in your app, you can extend the generated code
 - [Using WinTS to build WPF apps](./docs/WPF/getting-started-endusers.md)
 - [Working on WinTS](./docs/getting-started-developers.md)
 
+Note. When reading the documentation, some markdown files have not been copied to the dev branch, if you find a link is 404'ing change to the 'master' branch.
```

Comment:
The changes we did in dev will be merged into master once WPF template development is finished. 
Can we change that to somewhat like Docs related to UWP templates have been moved to the UWP folder? 
Also, could you point me to the broken link? We should fix that too.

Reply:
My apologies. I mis-read a link between ./uwp/services and './services', i'll roll back the readme update.

There is a broken link - I will track it down and report back.

Reply:
Awesome! Thank you!